extensions/smallrye-metrics/deployment/src/main/java/io/quarkus/smallrye/metrics/deployment/SmallRyeMetricsProcessor.java

```diff
@@ -202,15 +203,16 @@ AutoInjectAnnotationBuildItem autoInjectMetric() {
         return new AutoInjectAnnotationBuildItem(SmallRyeMetricsDotNames.METRIC);
     }
 
+    @BuildStep(providesCapabilities = Capabilities.METRICS)
```

Comment:
This is now deprecated through https://github.com/quarkusio/quarkus/pull/5195/files#diff-909be471039482910981e691b9083e56R77

Reply:
done